coral-dbt/src/main/resources/tests/test_incremental.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright 2023 LinkedIn Corporation. All rights reserved.
+# Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+#
+import subprocess
+import requests
+import unittest
+from jinja2 import Template
+
+SEEDS_DIR = 'seeds/'
+URL = 'http://localhost:8080/api/incremental/rewrite'
+
+class TestUtils:
+    @staticmethod
+    def construct_incremental_code_generation_test(name):
+        output_table = name + '_output'
+        expected_file = open(SEEDS_DIR + 'test_' + name + '_expected.txt', 'r')
+        expected_incremental_sql = expected_file.read()
+        expected_file.close()
+
+        return({
+            'output_table': output_table,
+            'expected_incremental_sql': expected_incremental_sql,
+        })
+
+    @staticmethod
+    def test_incremental_code_generation(coral_response, table_names, output_table):
+        with open('incremental_code_generation.j2') as f:
+            template = Template(f.read())
+
+        context = {'coral_response': coral_response, 'table_names': table_names, 'output_table': output_table}
+        output = template.render(context)
+
+        return output
+
+class TestIncremental(unittest.TestCase):
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(self):
+        subprocess.run(["chmod", "+x", "./setup_test.sh"])
```

Comment:
Why is this needed?

Reply:
The command is to make the `setup_test.sh` file executable.